packages/core/web-test-runner.config.mjs

```diff
@@ -45,5 +52,5 @@ export default /** @type {import("@web/test-runner").TestRunnerConfig} */ ({
       <script type="module" src="./web-test-runner.jasmine.js"></script>
     </head>
     <body></body>
-  </html>`
+  </html>`,
```

Comment:
Is this file alright with this trailing comma?

Reply:
I'll see if I can revert it. Prettier might have formatted it after my other changes ...

Reply:
Prettier reformats it on any change, even if I revert the changes. I could manipulate the system to comit it and turn prettier off, commit and then turn prettier back on and commit that change but imo I think it is probably ok and easier to keep it.

Reply:
That's fine. Just seemed weird.